src/main/python/pyserini.py

```diff
@@ -0,0 +1,103 @@
+from py4j.java_gateway import JavaGateway
+
+class Pyserini:
+    """Common base class for all methods accessing Anserini (Java)
+    Attributes
+    ----------
+    gateway : :obj:`JavaGateway`
+        The Java Gatewat object.
+    index : str
+        The directory path to the index.
+    pyserini : :obj:`PyseriniEntryPoint`
+        The entry point to the Java class PyseriniEntryPoint.
+    """
+
+    def __init__(self, index_path):
+        """
+           Constructor for the Pyserini class.
+
+           Parameters
+           ----------
+           index_path : str
+               The directory path for the Lucene index.
+        """
+        self.gateway = JavaGateway()
+        self.index = self.gateway.jvm.java.lang.String(index_path)
+        self.pyserini = self.gateway.jvm.io.anserini.py4j.PyseriniEntryPoint()
+        self.pyserini.initializeWithIndex(index_path)
+
+    def search(self, query_string, num_hits=20):
```

Comment:
Since this is a default argument I thought the standard is to have no space around `=`. See http://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value.